app/Models/ContestJudge.php

```diff
@@ -27,4 +28,17 @@ public function user(): BelongsTo
     {
         return $this->belongsTo(User::class, 'user_id');
     }
+
+    public function stdDev()
+    {
+        // TODO: treat missing scores as 0?
```

Comment:
all judges are supposed to judge all entries and they're poked if they haven't
default of 0 could make sense if it's to indicate that something is wrong, though ideally we'd know by counting the number of votes